functions-and-scope.js

```diff
@@ -16,8 +16,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+let numberOfEightOrHigher = 0;
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] > 7) {
+        numberOfEightOrHigher = numberOfEightOrHigher + 1;
```

Comment:
Dit kun je versimpelen:
```suggestion
        numberOfEightOrHigher++;
```